setup.cfg

```diff
@@ -83,6 +83,8 @@ addopts = --doctest-rst -m "not figure"
 markers =
     online: marks this test function as needing online connectivity.
     figure: marks this test function as using hash-based Matplotlib figure verification. This mark is not meant to be directly applied, but is instead automatically applied when a test function uses the @sunpy.tests.helpers.figure_test decorator.
+    flaky
+    array_compare
```

Comment:
What do these do?

Reply:
pytest raises a warning now if any markers used aren't in this list (presumably so users have a definitive list of markers?)

Reply:
Could you add some descriptions like the other two?

Reply:
Both of these are really fails on the plugin to not register their marks.